docs/adr/0048-notification-setting.md

```diff
@@ -0,0 +1,19 @@
+# 48. Notification setting
+
+Date: 2024-06-04
+
+## Status
+
+Accepted
+
+## Context
+
+- User need a way to tell and change notification configuration in-app
+- However, no app is allowed directly change the device's settings by design
+- With Gmail, Google will redirect users to system settings if they desire to change it
```

Comment:
we should compare with Samsung Mail app also

Reply:
My Android real device cannot install Samsung Mail, and I have no access to Samsung real device, so I cannot confirm its behavior.

Reply:
please find the way to compare with some other app. also other mail app in store.